larray/example.py

```diff
@@ -0,0 +1,43 @@
+import os
+import larray as la
+
+EXAMPLE_FILES_DIR = os.path.dirname(__file__) + '/tests/data/'
+
+def load_example_arrays():
```

Comment:
also, I never liked that hh array. We cannot combine it with the two others in anyway and it is not super interesting by itself, so I do not think it justifies its place in the tutorial.